src/components/new-request/search.vue

```diff
@@ -681,8 +698,8 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
 
 // Line spacing between bullet points and sizing
 .bullet-points {
-  font-size: $px-14;
   line-height: 1.5rem;
+  font-size: $px-15;
```

Comment:
Uxpin:

![image](https://github.com/bcgov/namerequest/assets/10002889/f23e2bae-c3ab-405d-9205-45ac64c95158)

Actual:

![image](https://github.com/bcgov/namerequest/assets/10002889/06b6b041-d856-4e82-b031-65ccae0b3785)

Reply:
We could probably go to 16px for the bullet font size, but UXA didn't say anything last time so let's see what they think here.